dmd_Jkk1.m

```diff
@@ -0,0 +1,25 @@
+function Jkk1=dmd_Jkk1(x,k,b)
```

Comment:
This function looks pretty much like it could be implemented in one line using the sum() command and and arrayfun or a bsxfun. This would make the file superfluous, things could be kept in distanceDiracGaussian()...
Optionally, one could implement it as a function within distanceDiracGaussian(). I think a file of its own is too much here...